dfastmi/gui/dialog_model.py

```diff
@@ -205,14 +206,15 @@ def check_configuration(
         config = self.get_configuration(branch, reach, reference_files, measure_files)
         return check_configuration(self.rivers, config)
 
-    def run_analysis(self) -> bool:
+    def run_analysis(self, gui: bool = False) -> bool:
```

Comment:
is this really necessary? why? This is part of a MVVM principle,  why wouldn't you use dfastmi.batch.core.batch_mode_core directly

Reply:
It is also not used in the code.. please remove

Reply:
This is added in relation to the generated figures.
When all figure are generated, this parameter is used to determine if the control of closing the figures should be returned to the user.

Reply:
Discussed with Ralph.

Reply:
See #103